Source/AwsCommonRuntimeKit/io/retryer/CRTAWSRetryStrategy.swift

```diff
@@ -2,114 +2,123 @@
 //  SPDX-License-Identifier: Apache-2.0.
 
 import AwsCIo
+import Foundation
 
-public final class CRTAWSRetryStrategy {
-    let allocator: Allocator
-
-    var rawValue: UnsafeMutablePointer<aws_retry_strategy>
-
-    init(retryStrategy: UnsafeMutablePointer<aws_retry_strategy>,
-         allocator: Allocator) {
-        self.rawValue = retryStrategy
-        self.allocator = allocator
-    }
+//TODO: rename class to RetryStrategy or CRTRetryStrategy. We have inconsistent CRT/AWS as a prefix of some classes.
+// I am not renaming it for now because it messes up the git change log. Will create a separate PR for just renaming.
```

Comment:
Food for thought: We don't need to use any prefixes (here and throughout this codebase). There won't be any name collisions given names just need to be unique within the same module.

Reply:
Thanks, I will remove the inconsistent prefix